src/crud.js

```diff
@@ -0,0 +1,72 @@
+// eslint-disable-next-line import/no-cycle
+import {
+  tasks, displayTaskList, storeDateInLocalStorage, getDateFromLocalStorage,
+} from './index.js';
+
+function addTask(description) {
+  const id = getDateFromLocalStorage().length;
+  const task = { description, complete: false, index: id + 1 };
+  tasks.push(task);
+  storeDateInLocalStorage(tasks);
+  displayTaskList();
+}
+
+function handleKeyPress(e) {
+  const inputValue = document.getElementById('input');
+  const description = inputValue.value;
+  if (e.key === 'Enter') {
+    e.preventDefault();
+    addTask(description);
+    inputValue.value = '';
+  }
```

Comment:
Write each function in a specific file that best describes the functionality. do this for your crud.js and index.js files, so your project is clean and clear 👍🏼.